### DISCUSSION TITLE
Does anyone have results they can share regarding backtest vs. live trading?

### DISCUSSION THREAD

**Discussion by crazy25000** [nautechsystems/nautilus_trader#331] [Q&A] [answered]
Hello,

Would anyone like to share their results about the accuracy, realism of the library?

For example, ran a strategy live for at least a week. Then downloaded historical data and run a backtest to compare the results to the live trades.

I'm finding myself rewriting my own framework in Cython more often for performance. Than discovered this library, got me excited about maybe switching :p

Thanks,

**Comment by limx0** [comment, marked as answer]
Hey @crazy25000, this is somewhat of a hard question to answer, but let me detail out a couple of things nautilus currently _doesnt_ do with relation to backtests: 
- It assumes you have 0 latency between sending orders and the exchange
- It assumes if you have a limit order on a price level, if that price trades, then your order is filled. 

I'd like to address these issues in the near future (weeks to months), but for now it depends on what sort of assumptions you're happy with. 

That being said, if you are only trading aggressively (fill or kill orders) and you aren't too concerned about how latency to the broker/exchange affects your orders, the backtest should very closely match what you would see running live in my opinion. Aside from the issues mentioned above, I think nautilus does a very good job of maintaining a _very_ similar backtest and live trading environment (the code paths for backtest vs live are very similar)

**Reply by crazy25000** [reply]
Thank you for the response! I don't have concerns about latency, haven't ran into issues with it before. I think providing result comparisons between backtests and live trading would be useful. I may just run it live with a demo account to test.
